### PR TITLE
Jpotter blog post

### PR DESCRIPTION
**Pull request by jmp1617**

### CODE REVIEW

File: _posts/2017-12-27-deepnet-synapse-delta-visualizer.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+Thanks to OpenGL, ( more specifically the OpenGL, OpenGL ES, and Vulkan development library GLFW, and GLEW for function pointer association ) you are able to watch the neural network learn. The fact that the base network is 4 layers is important for this implementation because that means that there are 3 synaptic weight matices, and therefor, allows me to easily map each synapse to a respective Cartesian coordinate axis (X,Y,Z). I then render the location of the normalized ( -1.0 to 1.0 ) synaptic weight values to the screen and redraw each frame without clearing so a path can be seen ( default mode ).
```
Comment (sgreene570) [inline _posts/2017-12-27-deepnet-synapse-delta-visualizer.md]: matrices, therefore

**Comment by sgreene570** [comment]
Can you squash your commits into 1 commit?

**Comment by jmp1617** [comment]
All set

**Comment by devinmatte** [comment]
I don't disagree that the post is good, I am just of the opinion that if this is a major project, that we wait until the project has passed before merging.

No big deal but maybe we could discuss/keep that in mind for the future

**Pull request merged**